util/src/test/resources/expected.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    app: mock
+  name: mock
+spec:
+  ports:
+  - port: 99
+    protocol: TCP
+    targetPort: 9949
+  selector:
+    app: mock
+---
+apiVersion: apps/v1beta1
+kind: Deployment
+metadata:
+  labels:
+    app: helloworld
+  name: helloworld
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: helloworld
+      name: helloworld
+    spec:
+      containers:
+      - args:
+        - -http=127.0.0.1:8080
+        image: gcr.io/hightowerlabs/helloworld:0.0.1
+        imagePullPolicy: Always
+        name: helloworld
+        resources:
+          limits:
+            memory: 1Gi
+            cpu: 500m
+          requests:
+            memory: 4Gi
+            cpu: '2'
```

Comment:
Not sure why the quotes are here. I guess it's because `QuantityFormatter.toBase10String()` returns a String?